tests/gds_kernel_latency.c

```diff
@@ -234,15 +237,15 @@ static struct pingpong_context *pp_init_ctx(struct ibv_device *ib_dev, int size,
                         ctx->txbufexp_addr = (uintptr_t*)gpu_malloc(page_size, sizeof(uintptr_t)*max_batch_len);
                 }
         } else {
-                ctx->txbuf = memalign(page_size, ctx->txtot_size); //posix_memalign
-                ctx->rxbuf = memalign(page_size, ctx->rxtot_size);
+                assert(0 == posix_memalign((void **)&(ctx->txbuf), page_size, ctx->txtot_size));
+                assert(0 == posix_memalign((void **)&(ctx->rxbuf), page_size, ctx->rxtot_size));
```

Comment:
careful with assert() as it is a no-op if NDEBUG is defined.

Reply:
I would rather not do this change in this PR, as it is not essential and unrelated to the prototype